xcp_d/workflow/postprocessing.py

```diff
@@ -216,6 +216,65 @@ def fwhm2sigma(fwhm):
     return fwhm / np.sqrt(8 * np.log(2))
 
 
+def init_pre_smoothing(mem_gb,
```

Comment:
Other than the description, is this workflow any different from `init_resd_smoothing`? If not, maybe `init_resd_smoothing` could be generalized into `init_smoothing`, and the description could be either overridden outside the workflow or toggled based on a new parameter within the workflow.